content/TruenasApps/StableApps/InstallNextCloudMedia.md

```diff
@@ -15,151 +15,284 @@ keywords:
+
+{{< expand "Earlier Nextcloud Deployment Datasets" "v" >}}
+Earlier deployments of the Nextcloud app use five datasets, the parent dataset for the application (**nextcloud**) and the four child datasets:
+* **appdata** that contains HTML, apps, custom_themes, config, etc.
+* **userdata** that contains the actual files uploaded by the user
+* **pgdata** that contains the database files.
+* **pgbackup** that contains the database backups
+
+Upgrading to 24.10 migrates earlier Nextcloud Kubernetes app deployments to the current Docker Compose configuration.
+{{< /expand >}}
+{{< include file="/static/includes/apps/InstallAppsStorageConfig2.md" >}}
+See **Mounting an SMB Share** below for details.
+
+{{< expand "Creating App Datasets" "v" >}}
+To create the Nextcloud app datasets, go to **Datasets**, select the dataset you want to use as the parent dataset, then click **Add Dataset** to [add a dataset]({{< relref "DatasetsScale.md" >}}).
+In this example, we create the Nextcloud datasets under the root parent dataset **tank**.
+
+Enter **nextcloud** in **Name**, and select **Apps** as the **Dataset Preset**.
+Click **Advanced Options** if you want to make any other setting changes. Click **Save**.
+When prompted, select **Return to Pool List** to configure permissions later after adding the other three datasets, or open the ACL editor to edit ACL permissions immediately after adding the dataset.
+
+Next, select the **nextcloud** dataset, and click **Add Dataset** to add the first child dataset.
+Enter **html** in **Name** and select **Apps** as the **Dataset Preset**.
+Click **Advanced Options** if you want to make any other setting changes. Click **Save**.
+
+Repeat this two more times to add the other child datasets to the **nextcloud** parent dataset.
+When finished you should have the **nextcloud** parent dataset with three child datasets under it. Our example paths are:
+* */mnt/tank/nextcloud/***html**
+* */mnt/tank/nextcloud/***data**
+* */mnt/tank/nextcloud/***postgres_data**
+
+{{< trueimage src="/images/SCALE/Apps/AppsAddNextcloudDatasets.png" alt="Add Nextcloud Storage" id="Add Nextcloud Storage" >}}
+{{< /expand >}}
+
+#### ACL and ACE Settings
+
+{{< include file="/static/includes/apps/AppInstallWizardACLConfiguration.md" >}}
+
+#### Mounting an SMB Share
+TrueNAS **Additional Storage** options include the ability to mount an SMB share inside the container pod.
+
+{{< include file="/static/includes/apps/AppWizardStorageSMBOption.md" >}}
+
+### Resource Configuration
+
+{{< trueimage src="/images/SCALE/Apps/InstallNextcloudResourcesConfig.png" alt="Resources Configuration Settings" id="Resources Configuration Settings" >}}
+
+{{< include file="/static/includes/apps/AppInstallWizardResourceConfig.md" >}}
+{{< include file="/static/includes/apps/AppsInstallWizardGPUPassthrough.md" >}}
 
 ## Troubleshooting Tips
 {{< hint type=info title="Update pre v. 2.0.4 Nextcloud Installations">}}
```

Comment:
Not touched in this PR, but we might need to clarify that this was about Dragonfish only.

Reply:
I'll remove this section. the 24.04 docs has the old tutorial with this content so it is still available in that branch.